recipes/yaml-cpp4rkt/bld.bat

```diff
@@ -0,0 +1,13 @@
+@REM Configure the build of phreeqc4rkt
+cmake -S . -B build ^
+    -DCMAKE_BUILD_TYPE=Release ^
+    -DCMAKE_PREFIX_PATH=%LIBRARY_PREFIX% ^
+    -DCMAKE_INSTALL_PREFIX=%LIBRARY_PREFIX% ^
+    -DBUILD_SHARED_LIBS=ON ^
+    -DYAML_BUILD_SHARED_LIBS=ON ^
+    -DYAML_CPP_BUILD_TESTS=OFF
+
+
+@REM Build and install phreeqc4rkt in %LIBRARY_PREFIX%
```

Comment:
```suggestion
@REM Build and install yaml-cpp4rkt in %LIBRARY_PREFIX%
```

Reply:
This is the danger of copying and pasting recipes! :)